src/i_video.c

```diff
@@ -1013,14 +1016,49 @@ static double CurrentAspectRatio(void)
 
     double aspect_ratio = (double)w / (double)h;
 
-    if (aspect_ratio > 2.4)
+    if (aspect_ratio > ASPECT_RATIO_MAX)
     {
-        aspect_ratio = 2.4;
+        aspect_ratio = ASPECT_RATIO_MAX;
     }
 
     return aspect_ratio;
 }
 
+// Fineangles in the SCREENWIDTH wide window.
+#define FIELDOFVIEW 2048
+
+void I_UpdateFOV(void)
+{
+    if (widescreen == RATIO_ORIG)
```

Comment:
Why not allow user-adjustable FOV at 4:3?

Reply:
You're right, I switched the logic around so it's possible now. Thanks! https://github.com/fabiangreffrath/woof/pull/1364/commits/dc894fd8fe79cd4803d06942327e70f8397f5861